### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1154] [Galaxy Trivia] [answered]
<img width="281" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/559e1f07-4028-4c53-ba79-06bfcd12f7b5">

# Objective / Rules
 - Fill in the grid using numbers from 1 to 9.
 - Each number should be used only once.
 - The three vertical and three horizontal equations must be true.

**Comment by tomerf** [comment, marked as answer]
| 9 | - | 3  | X | 5  | = | -6 |
|---|--:|----|---|----|---|----|
| + |   | +  |   | +  |   |    |
| 4 | + | 1  | + | 7  | = | 12 |
| - |   | X  |   | X  |   |    |
| 6 | X | 8  | - | 2  | = | 46 |
| = |   | =  |   | =  |   |    |
| 7 |   | 11 |   | 19 |   |    |
